### DISCUSSION TITLE
Welcome to ESPAltherma Discussions!

### DISCUSSION THREAD

**Discussion by raomin** [raomin/ESPAltherma#142] [Announcements]
## 👋 Welcome to ESPAltherma discussions!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Denele1984** [comment]
Good afternoon  Please tell me if the heat pump EBHQ006BAV3 supports both heating and cooling, and the control board came only for heating EKCBH008BBV3, is it possible to make it work for cooling as well?
